src/Widgets/ImageWidget/DimensionsEditor.scss

```diff
@@ -111,5 +183,35 @@
       font-size: 13px;
       color: #eee;
     }
+
+    .enum_attribute {
+      button {
+        border-left: 30px solid rgba(0, 0, 0, 0.1);
+        background: rgba(222, 222, 222, 0.2);
+        color: #eee;
+        span {
+          color: #eee;
+        }
+      }
+
+      button.enum_attribute_active {
```

Comment:
FYI: With `&.enum_attribute_active` this could be inlined into the `button {` section (as done below for `:disabled` as well).